### PR TITLE
Setup Gin dependencies and packages

### LINKED ISSUE #1: Setup Gin dependencies and packages
## Why
Setup Gin dependencies and packages for API projects.

## Acceptance Criteria
- Setup necessary dependencies and packages for API projects.
- Able to call simple API.

### PR DESCRIPTION
**Pull request by gutakk**
## What happened 👀
Resolve #1 
- Install Gin.
- Add API V1 endpoint which is `/api/v1/health`.
- Add Makefile with start development server command.
- Add `routers` directory inside api version because I think each api version should have their own router and combine all of them in bootstrap.
- Remove `bootstrap.go` from cmd as I think `main.go` can directly call function from `bootstrap` directory.
 
## Insight 📝
**Project generation steps**
- Install cookiecutter (you can follow the instruction in README).
- Checkout to this branch.
- Run command `cookiecutter <path to this repo in your local machine>`
- Your new application is created 🎉 .

## Proof Of Work 📹
- Start development server using `make dev` (server running at port 8080)
Screen Shot 2564-02-10 at 15 33 20

- Call API `localhost:8080/api/v1/health`
```http
# Request
curl --location --request GET 'localhost:8080/api/v1/health'

# Response
200 OK
{
    "status": "alive"
}
```

### CODE REVIEW

**Comment by carryall** [comment]
@gutakk Do we plan to keep the health API as a default API for the template or we decided to remove it later? Also not sure about having a separate health API for each API version 🤔

**Comment by gutakk** [comment]
> @gutakk Do we plan to keep the health API as a default API for the template or we decided to remove it later? Also not sure about having a separate health API for each API version 🤔

I planned to keep `health` API as a default API for the template. I want to give the example to the developers who start using our template and they can delete the Health API related stuffs (controller and router in this case).
How about add some comment at the Health API code something like `This is an example API, you can delete it`?

**Review by olivierobert — APPROVED** [review]
Looks good to me 👍

File: {{cookiecutter.app_name}}/README.md [hunk 1]
```diff
@@ -12,7 +12,10 @@
 
 ### Development
 
-*Development instruction goes here ...*
+- Start development server
```
Comment (junan) [inline {{cookiecutter.app_name}}/README.md:15]: @gutakk Could we add an empty space? We usually add an empty line before code snippet 
```suggestion
- Start development server

```

**Review by junan — APPROVED** [review]

File: {{cookiecutter.app_name}}/README.md (hunk 1 above)
Comment (gutakk) [reply {{cookiecutter.app_name}}/README.md:15]: Absolutely. Fixed in c264c5147bfd5692f0233518ba2c18ffc84c0c43

**Review by carryall — APPROVED** [review]

**Comment by gutakk** [comment]
@carryall Added comment in 83788e0ae0b89ed401973329487d85bd9ee6be7e

**Pull request merged**